packages/admin-panel/src/table/columnTypes/generateConfigForColumnType.jsx

```diff
@@ -51,22 +51,25 @@ const BUTTON_COLUMN_TYPES = [
   'qrCode',
   'testDatabaseConnection',
   'bulkEdit',
+  'sync',
 ];
 
 export const generateConfigForColumnType = (type = 'tooltip', actionConfig, reduxId) => {
   const CustomCellComponent = CUSTOM_CELL_COMPONENTS[type];
   if (!CustomCellComponent) {
     return {};
   }
-  let config = {
-    Cell: generateCustomCell(CustomCellComponent, actionConfig, reduxId),
-  };
+
   if (BUTTON_COLUMN_TYPES.includes(type)) {
-    config = {
-      ...config,
+    return {
+      Cell: generateCustomCell(CustomCellComponent, actionConfig, reduxId),
       ...BUTTON_COLUMN_OPTIONS,
     };
   }
+  let config = {
+    Cell: generateCustomCell(CustomCellComponent, actionConfig, reduxId),
```

Comment:
Considering this `Cell` definition is used in two places, is it worth putting this into variable?